### PR TITLE
Cargo publish and changelog generation

### PR DESCRIPTION
**Pull request by metaclips**
## Proposed Changes
Indicate if a crate is to be published in Cargo.toml

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://www.ockam.io/learn/how-to-guides/contributing/CONTRIBUTING#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://www.ockam.io/learn/how-to-guides/high-performance-team/conduct/).
- [x] I have accepted the Ockam [Contributor Licence Agreement](https://www.ockam.io/learn/how-to-guides/contributing/cla/) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/ockam-network/contributors/blob/master/CONTRIBUTORS.csv) file in a separate pull request to the [ockam-network/contributors](https://github.com/ockam-network/contributors) repository.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name = "ockam_command"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+publish = true
```
Comment (jared-s) [inline implementations/rust/ockam/ockam_command/Cargo.toml:6]: `ockam_command` isn't ready to be published, can you set this to `false` for now please?

**Review by jared-s — CHANGES_REQUESTED** [review]
Looking fantastic. Just one change: not yet publishing ockam_command

**Review by jared-s — APPROVED** [review]

**Comment by jared-s** [comment]
@metaclips Anything else you want to add to this branch, or shall I merge it? I'd like to try out the scripts for today's release

**Comment by metaclips** [comment]
> @metaclips Anything else you want to add to this branch, or shall I merge it? I'd like to try out the scripts for today's release

It's a go @jared-s

**Comment by jared-s** [comment]
Already merged as #2314

**Pull request closed without merging**